package.json

```diff
@@ -10,14 +10,20 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "jest --notify --config=jest.config.json"
+    "test": "jest --notify --config=jest.config.js"
```

Comment:
[Q] Do you need to declare a config isn't it auto-magically found?

Reply:
Probably would be, will confirm